cmd/config.go

```diff
@@ -114,22 +106,42 @@ func defaultConfig(keyHome string, debug bool) []byte {
 		modules = append(modules, v)
 	}
 	cfg := Config{
-		Chain: &client.ChainClientConfig{
-			Key:            "default",
-			ChainID:        "cosmoshub-4",
-			RPCAddr:        "https://cosmoshub-4.technofractal.com:443",
-			GRPCAddr:       "https://gprc.cosmoshub-4.technofractal.com:443",
-			AccountPrefix:  "cosmos",
-			KeyringBackend: "test",
-			GasAdjustment:  1.2,
-			GasPrices:      "0.01uatom",
-			KeyDirectory:   keyHome,
-			Debug:          debug,
-			Timeout:        "20s",
-			OutputFormat:   "json",
-			BroadcastMode:  "block",
-			SignModeStr:    "direct",
-			Modules:        modules,
+		DefaultChain: "cosmoshub",
+		Chains: map[string]*client.ChainClientConfig{
+			"cosmoshub": {
+				Key:            "default",
+				ChainID:        "cosmoshub-4",
+				RPCAddr:        "https://cosmoshub-4.technofractal.com:443",
+				GRPCAddr:       "https://gprc.cosmoshub-4.technofractal.com:443",
+				AccountPrefix:  "cosmos",
+				KeyringBackend: "test",
+				GasAdjustment:  1.2,
+				GasPrices:      "0.01uatom",
+				KeyDirectory:   keyHome,
+				Debug:          debug,
+				Timeout:        "20s",
+				OutputFormat:   "json",
+				BroadcastMode:  "block",
+				SignModeStr:    "direct",
+				Modules:        modules,
+			},
+			"osmosis": {
+				Key:            "default",
+				ChainID:        "osmosis-1",
+				RPCAddr:        "https://osmosis-1.technofractal.com:443",
+				GRPCAddr:       "https://gprc.osmosis-1.technofractal.com:443",
+				AccountPrefix:  "osmo",
+				KeyringBackend: "test",
+				GasAdjustment:  1.2,
+				GasPrices:      "0.01uosmo",
+				KeyDirectory:   keyHome,
+				Debug:          debug,
+				Timeout:        "20s",
+				OutputFormat:   "json",
+				BroadcastMode:  "block",
+				SignModeStr:    "direct",
+				Modules:        modules,
+			},
```

Comment:
Do we need these any more?

Reply:
I like populating a default config with some stuff. Are you opposed to keeping this?

Reply:
I'd rather it get populated from the registry :) 

```
                        defaults := ["cosmoshub", "osmosis"]
			for _, chain := range defaults {
				ch, err := getChainConfigFromRegistry(chain, path.Join(home, "keys"), debug)
				if err != nil {
					return err
				}
				config.Chains[chain] = ch
			}

```